index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width">
+        <meta name="author" content="Amos Gloria">
+        <title>Facebook Clone | Welcome</title>
+        <link rel="stylesheet" href="./css/style.css">
+        </head>
+        <body>
+            <header>
+                <div class="header-block">
+                    <div id="branding">
+                        <h1><span class="highlight">facebook</span></h1>
```

Comment:
The `highlight` class is not referenced in your `style.css` for the `facebook` logo. Reference it and change the font to align with the given design